### ISSUE TITLE
"Warning: no department given" when running with rubocop 0.75.0

### ISSUE THREAD

**Issue by duncanjbrown** [alphagov/govuk-lint#127]
We use this gem by including the following lines at the head of our `.rubocop.yml`:

```
inherit_gem:
  govuk-lint: configs/rubocop/all.yml
```

We just upgraded to `rubocop v0.75.0`, which has just been released. Running `bundle exec rubocop` now produces many screenfuls of warnings like this before the (correct) linter output.

```
/Users/user/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/govuk-lint-4.0.0/configs/rubocop/gds-ruby-styleguide.yml: Warning: no department given for BlockAlignment.
/Users/user/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/govuk-lint-4.0.0/configs/rubocop/gds-ruby-styleguide.yml: Warning: no department given for CaseIndentation.
/Users/user/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/govuk-lint-4.0.0/configs/rubocop/gds-ruby-styleguide.yml: Warning: no department given for ElseLayout.
/Users/user/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/govuk-lint-4.0.0/configs/rubocop/gds-ruby-styleguide.yml: Warning: no department given for EmptyLineBetweenDefs.
/Users/user/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/govuk-lint-4.0.0/configs/rubocop/gds-ruby-styleguide.yml: Warning: no department given for EmptyLines.
/Users/user/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/govuk-lint-4.0.0/configs/rubocop/gds-ruby-styleguide.yml: Warning: no department given for EndAlignment.

<snip ~160 more lines>
```

I think it's complaining that many of the cop configurations in `govuk-lint` aren't fully-qualified with respect to "department", ie `Metrics`, `Lint`, etc. For instance, editing the `configs/rubocop/gds-ruby-styleguide.yml` to add `Layout` in front of `BlockAlignment` makes the first warning in the above list disappear.

It's not clear from the `rubocop` [changelog](https://github.com/rubocop-hq/rubocop/releases/tag/v0.75.0) what caused this.

**Comment by issyl0** [comment]
Hi @duncanjbrown, thanks for raising this issue.

We've been seeing this with that version of Rubocop too, since yesterday.

Thanks for the pointer towards a fix - I'm working on doing the same for all our cops now.

**Comment by issyl0** [comment]
We've just released govuk-lint 4.0.1, now available on RubyGems, with the fix for this issue. Thanks!

**Comment by duncanjbrown** [comment]
Thank you!
